testsupport/tiers/checks.go

```diff
@@ -1395,6 +1397,18 @@ func gitOpsServiceLabel() namespaceObjectsCheck {
 	}
 }
 
+func appstudioWorkSpaceNameLabel() namespaceObjectsCheck {
+	return func(t *testing.T, ns *corev1.Namespace, memberAwait *wait.MemberAwaitility, _ string) {
+		// TODO fix for migration/existing namespaces cases
+		labelWaitCriterion := []wait.LabelWaitCriterion{}
+		if !strings.HasPrefix(ns.Name, "migration-") {
+			labelWaitCriterion = append(labelWaitCriterion, wait.UntilObjectHasLabel("appstudio.redhat.com/workspace_name", ns.Name))
+		}
```

Comment:
@rajivnathan correct me if I'm wrong, we don't need this check anymore - we take care of updating the label for the already existing namespaces, right? 
I know that you introduced this check for the other label https://github.com/codeready-toolchain/toolchain-e2e/pull/494/commits/e71b580164777834e1086ace135b1dab4c4c6a8d but it was before we fixed that in the NSTemplateSet controller https://github.com/codeready-toolchain/member-operator/pull/372 so the already existing namespaces should be updated now.

```suggestion
		labelWaitCriterion = append(labelWaitCriterion, wait.UntilObjectHasLabel("appstudio.redhat.com/workspace_name", ns.Name))
```

Reply:
ack

Reply:
Actually seems that the `TestAfterMigration` is failing because the namespace `migration-appstudio-provisioned-tenant` doesn't have that expected label. 
So probably that if statement is still needed.

Reply:
yes, added them back. let's see.

Reply:
if it failed, then it means that there is a bug in our logic. The already existing namespaces should be updated with the new label.

Reply:
Ok, I found what is wrong. The namespace was updated, but you expect a wrong value.
```
Actual labels:map[appstudio.redhat.com/workspace_name:migration-appstudio-provisioned 
```
vs:
```
appstudio.redhat.com/workspace_name:migration-appstudio-provisioned-tenant
```
@bamachrn you need to fix it in your tests - the label contains the name of the workspace, not of the namespace. And please drop the if-statement again.